plugins/bundle/plugin.go

```diff
@@ -664,3 +657,26 @@ func (*fileLoader) ClearCache() {
 func (*fileLoader) SetCache(string) {
 
 }
+
+func (fl *fileLoader) Trigger(ctx context.Context) {
+	fl.oneShot(ctx)
+}
+
+func (fl *fileLoader) oneShot(ctx context.Context) {
+	var u download.Update
+	u.Metrics = metrics.New()
+	f, err := os.Open(fl.path)
+	u.Error = err
+	if err == nil {
+		defer f.Close()
+		b, err := bundle.NewReader(f).
+			WithMetrics(u.Metrics).
+			WithBundleVerificationConfig(fl.bvc).
+			WithSizeLimitBytes(fl.sizeLimitBytes).Read()
+		u.Error = err
+		if err == nil {
+			u.Bundle = &b
+		}
+	}
```

Comment:
[nit] Since we're aborting on error, we could make that a bit more explicit:
```suggestion
	if err != nil {
		return
	}
	defer f.Close()
	b, err := bundle.NewReader(f).
		WithMetrics(u.Metrics).
		WithBundleVerificationConfig(fl.bvc).
		WithSizeLimitBytes(fl.sizeLimitBytes).Read()
	u.Error = err
	if err != nil {
		return
	}
	u.Bundle = &b
```

What do you think? Whenever I read `if err == nil` there's a bit of suspense building, like, _💭 "what if it isn't...?"_ and then it's a bit of a let-down if nothing follows 😄

Reply:
Re-organized this code a little. LMKWYT